src/app.ts

```diff
@@ -205,7 +207,31 @@ export default class ChessGame {
 			.then(value => this.preloads['check-marker'] = value));
 		await Promise.all(preloads);
 	}
+	
+	private async loadActorsAndEvents() {

```

Comment:
There's a lot of code duplication here with the contents of the `started` method. I think you may be able to just call `this.resetGame()` in started now. Something like:
```ts
private started = async () => {
    this.resetGame();
}
```

Reply:
Just pushed the changes. I also moved the resetGame() function closer to the top, near started()